cranelift/codegen/src/isa/aarch64/lower.isle

```diff
@@ -2515,7 +2515,7 @@
 
 ;; `targets` contains the default target with the list of branch targets
 ;; concatenated.
-(rule (lower_branch (br_table idx _ _) targets)
+(rule (lower_branch (br_table idx _) targets)
```

Comment:
Maybe cleanup for a followup PR, but is a separate targets argument still necessary?

Reply:
Right now it is, but I'd like to change that as well. It would be great to have a function for translating the blocks in the branch instructions instead of passing the labels as an extra argument.